zokrates_core/src/proof_system/mod.rs

```diff
@@ -21,6 +22,18 @@ impl SetupKeypair {
     }
 }
 
+impl Serialize for SetupKeypair {
```

Comment:
Any reason `derive(Serialize)` cannot be used here?

Reply:
No, I removed the unnecessary serializer